rtfs.go

```diff
@@ -94,10 +107,7 @@ func (im *IpfsManager) NewObject(template string) (string, error) {
 // Pin is a wrapper method to pin a hash.
 // pinning prevents GC and persistently stores on disk
 func (im *IpfsManager) Pin(hash string) error {
-	if err := im.shell.Pin(hash); err != nil {
-		return fmt.Errorf("failed to pin '%s': %s", hash, err.Error())
-	}
-	return nil
+	return im.shell.Pin(hash)
```

Comment:
why remove the more detailed error message?

Reply:
I wanted to bubble up the error message that the ipfs node was sending. I think as `rtfs` is being used as a library, it's best to send up raw error messages from the ipfs node, and then let importing services (ie temporal) handle the formatting of a detailed error message.

Reply:
that's fair 👍